numpyro/distributions/constraints.py

```diff
@@ -122,8 +122,11 @@ def __call__(self, x):
 
 class _PositiveDefinite(Constraint):
     def __call__(self, x):
+        # check for symmetric
+        symmetric = np.all(np.all(x == np.swapaxes(x, -2, -1), axis=-1), axis=-1)
         # check for the smallest eigenvalue is positive
-        return np.linalg.eigh(x)[0][..., 0] > 0
+        positive = np.linalg.eigh(x)[0][..., 0] > 0
+        return symmetric & positive
```

Comment:
I saw some tests are passed for the newest version so I remove the marks. By default, `np.linalg.eigh` uses the lower triangular part to compute eigen values, so we need to add a check for symmetric.